lib/category_select.dart

```diff
@@ -0,0 +1,182 @@
+
+import 'package:flutter/material.dart';
+import 'package:nexpot/main.dart';
+
+class CategorySelect extends StatelessWidget {
+  // ロジック組み込む際使用する
+  final List<String> categoryList = <String> [
+    '歴史',
+    '自然',
+    'グルメ',
+    'ショッピング',
+    'レジャー'
+  ];
+
+  final List<String> imageList = <String> [
+    'images/sightseeing-user-category-select/castle.png',
+    'images/sightseeing-user-category-select/tree.png',
+    'images/sightseeing-user-category-select/gourmet.png',
+    'images/sightseeing-user-category-select/cart.png',
+    'images/sightseeing-user-category-select/parasol.png',
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: Center(
+        child: Column(
+          children: <Widget>[
+            TitleArea(),
+            Flexible(
+                child: ListView(
+                  children: [
+                    ListArea(
+                      context,
+                      '歴史',
+                      'images/sightseeing-user-category-select/castle.png',
+                      true
+                    ),
+                    ListArea(
+                        context,
+                        '自然',
+                        'images/sightseeing-user-category-select/tree.png',
+                        false
+                    ),
+                    ListArea(
+                        context,
+                        'グルメ',
+                        'images/sightseeing-user-category-select/gourmet.png',
+                        false
+                    ),
+                    ListArea(
+                        context,
+                        'ショッピング',
+                        'images/sightseeing-user-category-select/cart.png',
+                        false
+                    ),
+                    ListArea(
+                        context,
+                        'レジャー',
+                        'images/sightseeing-user-category-select/parasol.png',
+                        false
+                    ),
+                  ],
+                )
+            ),
+            NextButton(context),
+          ],
+        ),
+      ),
+    );
+  }
+
+}
+
+class HexColor extends Color {
+  static int _getColorFromHex(String hexColor) {
+    hexColor = hexColor.toUpperCase().replaceAll('#', '');
+    if (hexColor.length == 6) {
+      hexColor = 'FF' + hexColor;
+    }
+    return int.parse(hexColor, radix: 16);
+  }
+
+  HexColor(final String hexColor) : super(_getColorFromHex(hexColor));
+}
+
+
+Widget TitleArea() {
+  return Container(
+    padding: EdgeInsets.only(
+        top: 120.0, right: 0.0, bottom: 5.0, left: 0.0
+    ),
+    child:Text(
+      '興味あるものを教えて下さい',
+      style: TextStyle(
+          color: HexColor('1E234D'),
+          fontSize: 20,
+          fontWeight: FontWeight.bold
+      ),
+    ),
+  );
+}
```

Comment:
[should]
1. 三ヶ所ぐらい使用すると思うので、デザインはそのままでtitleだけ受け取れるようにお願いします。
2. lib/widgetフォルダを作成してtitle_area.dartを作成して移動してください。
3. 部品にはpadding又はmarginを持たせないようしてください。⇨部品を使用するページ側で基本的に間隔を調整します。

```suggestion
Widget TitleArea(title) {
  return Container(
    child:Text(
      title,
      style: TextStyle(
          color: HexColor('1E234D'),
          fontSize: 20,
          fontWeight: FontWeight.bold
      ),
    ),
  );
}
```

Reply:
@illbe19 @damono999 

指摘内容に問題ありません、`lib/widget`配下に部品作成おねがいします！

（菅さんのフロントエンド技術が活きてる指摘ですね、、ここらへん私知見が疎いので助かります。）

Reply:
lib/widgetにファイルを作成して、そこから読み込むように修正いたしました。

Reply:
確認しました！

対応ありがとうございます！